Azrux/Athenas-front
# chore: update dependencies

## Description
`package.json` dependencies updated and lint command fixed

## How to test?
* Run `npm run lint` and check if the lint command is working fine
* Run `npm run dev` and check if the app is working fine
